Tests/Marketplace/Tests/marketplace_services_test.py

```diff
@@ -88,7 +88,7 @@ def test_validate_all_fields_of_parsed_metadata(self, dummy_pack, dummy_pack_met
         assert parsed_metadata['currentVersion'] == '2.3.0'
         assert parsed_metadata['versionInfo'] == "dummy_build_number"
         assert parsed_metadata['commit'] == "dummy_commit"
-        assert parsed_metadata['tags'] == ["tag number one", "Tag number two", "Use Case"]
+        assert set(parsed_metadata['tags']) == {"tag number one", "Tag number two", "Use Case"}
```

Comment:
add validation for the list length too, in the new form there could be duplicate tags

Reply:
I used set to build the tags list so no duplicates are expected.